src/Mod/Sketcher/Gui/CommandCreateGeo.cpp

```diff
@@ -5064,7 +5063,7 @@ namespace SketcherGui {
 class DrawSketchHandlerFillet: public DrawSketchHandler
 {
 public:
-    DrawSketchHandlerFillet() : Mode(STATUS_SEEK_First), firstCurve(0) {}
+    DrawSketchHandlerFillet(int filletType) : filletType(filletType), Mode(STATUS_SEEK_First), firstCurve(0) {}

```

Comment:
Could be use an enum instead the int filletType?

The rationale is to avoid the spread of "magic numbers" in code (and yes, I am aware that iMsg is an int). An enum tag is easier to read and understand and less error prone.